cmake/modules/AddSwift.cmake

```diff
@@ -536,7 +536,7 @@ endfunction()
 #
 # source1 ...
 #   Sources to add into this library
-function(_add_swift_library_single target name)
+function(_add_swift_library_single target module_target name)
```

Comment:
Sure, will add documentation.

I can try out adding a parse argument.  Will need to also change 'target' above as I only added 'module_target' in this way to be consistent.

Would you prefer that change first, then land the rest of this?